enterprise_access/apps/subsidy_access_policy/subsidy_api.py

```diff
@@ -30,6 +30,38 @@ class TransactionPolicyMismatchError(Exception):
     """
 
 
+def subsidy_learner_aggregate_data_cache_key(subsidy_uuid, policy_uuid=None):
+    return versioned_cache_key('get_subsidy_learners_aggregate_data', subsidy_uuid, policy_uuid)
+
+
+def get_and_cache_subsidy_learners_aggregate_data(subsidy_uuid, policy_uuid=None):
+    """
+    Get aggregated learner data for a given subsidy. This can be optionally further filtered
+    """
+    cache_key = subsidy_learner_aggregate_data_cache_key(subsidy_uuid, policy_uuid)
+    cached_response = request_cache(namespace=REQUEST_CACHE_NAMESPACE).get_cached_response(cache_key)
+    if cached_response.is_found:
+        logger.info(
+            f'subsidy_learners_aggregate_data cache hit for subsidy {subsidy_uuid} and policy {policy_uuid}'
+        )
+        return cached_response.value
+
+    client = get_versioned_subsidy_client(version=1)
```

Comment:
Could we just expose this new endpoint in version 2 of the API as well?

Reply:
scratch that, that's a different chunk of work to address

Reply:
talked offline and decided that a larger migration of policy related views was needed and that's slightly beyond the scope of this ticket/work and TBD